src/Controller.js

```diff
@@ -0,0 +1,119 @@
+const {
+  gameStart,
+  bridgeSize,
+  moving,
+  gameCommand,
+  gameMap,
+  gameResult,
+} = require("./UI/View");
+const {
+  APPROPRIATE_INPUT,
+  ANSWER,
+  REPLACEMENT,
+  GAME_MESSAGE,
+} = require("./Utils/Constants");
+const { UP, RESTART, QUIT } = APPROPRIATE_INPUT;
+const { RIGHT, WRONG, UNCHOSEN } = ANSWER;
+const { COMMA, LINE } = REPLACEMENT;
+const { SUCCESS, FAIL } = GAME_MESSAGE;
+const { close } = require("./Utils/MissionUtils");
+
+const BridgeGame = require("./BridgeGame");
+const BridgeMaker = require("./BridgeMaker");
+const BridgeRandomNumberGenerator = require("./BridgeRandomNumberGenerator");
+
+class Controller {
+  #bridge;
+  #round = 0;
+  #tries = 1;
+  #userState = { top: [], bottom: [] };
+
+  constructor() {
+    this.bridgeGame = new BridgeGame();
+  }
+
+  start() {
+    gameStart();
+    bridgeSize(this.generateBridge.bind(this));
```

Comment:
```suggestion
bridgeSize(this.generateBrige.bind(this,
 moving(this.passOrFail.bind(this))
    )
  )
```

이렇게 쓸 수 있도록 수정해보시는건 어떨까요?:)